### PR TITLE
New guide Compatible semantic tags

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?

Add new guide to list compatible semantic tags

### Motivation

https://datadoghq.atlassian.net/browse/DOCS-5202

### Additional Notes

Ready to merge after review

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by drichards-87** [comment]
Created DOCS-5373 for the editorial review.

File: content/en/dashboards/guide/compatible_semantic_tags.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Overview
+
+For compatible series of data, Datadog can map colors to meaning. When a compatible tag is detected, Datadog suggests the Semantic color palette. This automatically maps data to meaning-driven colors.
+
+**Note**: Queries must be grouped with a single set of tags.
+
+### Map compatible tags to colors based on their meaning
+
+For example, an error status code is mapped to red or success is green.
```
Comment (kayayarai) [inline content/en/dashboards/guide/compatible_semantic_tags.md:18]:
```suggestion
For example, an error status code is mapped to red, and success to green.
```

File: content/en/dashboards/guide/compatible_semantic_tags.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- link: "/dashboards/guide/widget_colors/#categorical-palettes"
+  tag: "Documentation"
+  text: "Selecting the right colors for your graphs"
+---
+
+## Overview
+
+For compatible series of data, Datadog can map colors to meaning. When a compatible tag is detected, Datadog suggests the Semantic color palette. This automatically maps data to meaning-driven colors.
+
+**Note**: Queries must be grouped with a single set of tags.
```
Comment (kayayarai) [inline content/en/dashboards/guide/compatible_semantic_tags.md:14]:
```suggestion
**Note**: To use the Semantic color palette, a query must be grouped with a single set of tags.
```

File: content/en/dashboards/guide/compatible_semantic_tags.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+{{< img src="/dashboards/guide/compatible_semantic_tags/semantic_option.png" alt="Semantic color option in the graph editor" style="width:100%;" >}}
+
+### Ensure consistent coloring across charts
+
+Charts with a semantic palette use the same, stable color for each tag. This allows you to easily trace a given tag across different graphs.
+
+### Grouping behavior
+
+Queries with a single set of tags are supported. If multiple groupers are used with the semantic palette, coloring will be consistent, but not meaning-driven.
```
Comment (kayayarai) [inline content/en/dashboards/guide/compatible_semantic_tags.md:28]:
```suggestion
Queries grouped with a single set of tags are supported. If multiple groupers are used with the semantic palette, coloring is consistent, but not meaning-driven.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**